api/gateway.proto

```diff
@@ -291,17 +291,22 @@ message GatewayConnectionStats {
   repeated SubBand sub_bands = 10;
 }
 
-// The plain text value of the gateway secret.
-message GatewaySecretPlainText{
-  map<string,bytes> values = 1 [(validate.rules).map.keys.string = {pattern: "^[a-z0-9](?:[-]?[a-z0-9]){2,}$" , max_len: 36}];
+// The plain text value of the Basic Station LNS gateway secret.
+message GatewayLBSLNSSecretPlainText{
+  string token_key = 1;
```

Comment:
Probably needs a length cap to avoid people storing megabytes of `token_key`

Reply:
Our API Keys are 98 characters so I've capped this at 128 characters.